javaslang/src/main/java/javaslang/Value.java

```diff
@@ -516,7 +516,7 @@ default CharSeq toCharSeq() {
      */
     @SuppressWarnings("unchecked")
     @GwtIncompatible("reflection is not supported")
-    default T[] toJavaArray(Class<T> componentType) {
+    default T[] toJavaArray(Class<? super T> componentType) {
```

Comment:
Yes, but how DO we make it work for e.g. `Lazy<Integer>` otherwise?
I think we should only revert, if we can find a better solution...